pandas/io/tests/parse_dates_tests.py

```diff
@@ -0,0 +1,471 @@
+"""
+Tests date parsing functionality for all of the
```

Comment:
The tricky part (compared to `tests/indexes`) is that there are really "three" base classes (`CParserHighMemory`, `CParserLowMemory`, `PythonParser`).  How does one organize that then?